index.html

```diff
@@ -42,5 +42,177 @@ <h1 class="name"> Hey there <br>
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="pictureholder-container">
+            <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="pictureholder-container">
+          <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      </section>
+        <section class="about-container">
+          <div class="text-aline">
+            </div>
+            <div class="text-about">
+              <div class="plane-container">
+                <h3>About me</h3>
+                <div class="plane-image">
+                  <img src="Images 2/shape.svg" alt="plane">
+                </div>
+              </div>
+              <div class="test2">
+                <div>
+                  <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                </div> 
+              </div>
+                <div class="button2">
+                  <button class="button-resume" type="button">Get my resume </button>
+                </div>
+            </div>
+            <div class="plane">
+              <img src="Images 2/shape2.svg" alt="square">
+            </div>
+          </div>
+           <div class="section6">
+            <article class="qoute">
+              <div class="holder-text">
+                <img src="Images 2/placeholder2.svg" alt="holder">
+                <div class="holder-text1">
+                  <img src="Images 2/language-icon.svg" alt="holder">
```

Comment:
Great job on the icon! It looks good. However, there is one improvement that can be made to align with the design:

- [ ] [Optional] Instead of using an image for the box shadow effect on the icon, please consider using CSS styles to add the box shadow. This will allow for more flexibility and easier customization in the future.

By implementing the box shadow effect using CSS, you will ensure that the icon matches the design specifications and maintain consistency throughout the project. Keep up the good work!